.github/workflows/container-image.yml

```diff
@@ -1,4 +1,4 @@
-name: Build container image
+name: Build and release policy server
```

Comment:
(opening thread here randomly)

> @viccuad , in order to have a more complete SBOM file, the tool needs access to the files generated by the build. Thus, the release workflow needs to have access to the target directory. That's the reason I've merged the release worflows into a single file. I can split it again and call the same worflows to build the binaries in both workflows (ci and release). Do you prefer that? One cons of this approach is that the build will run twice when a tag is released. What do you think?

But, with that, it means that the release job is run twice, possibly making 2 releases, or making 1 and then overwriting. I think we shouldn't do this, releases should be built reproducibly and it needs to be possible to automatically subscribe and consume then nicely (instead of getting 2 notifications for them per 1 real release).

As I understand, the SBOM creation needs to happen in the build job. I would like to have 1 workflow that:
1. builds the binary, generates the SBOM, signs both.
2. builds the container image, generates SBOM of container image (maybe with [`docker sbom`](https://docs.docker.com/engine/sbom/)), signs both.

And then the release workflow that we have, where we just download all the previous artifacts, and create a new GH release if there's a tag.

Reply:
We will not get additional notification. We will continue to have two as the current version. One for the job witch creates the release and another for the job witch build the container image. 

What will run twice is the container build process. Once for the `latest` tag and another for the `v*` tag. Each one in a different job. Witch will consume more resources and time and I would like to avoid. But at the time of the written I could not find a easy way to share artifacts between two different jobs as our current version is. I do not want to add to much complicated steps to make the communication works, keeping it simple and generating always the same results. Even if the push multiple commits in to main while releasing a tag. 

Let me see if I can find something else... 

PS: The current version of this PR already have some changes testing some changes since the first comment that originate this discussion.

Reply:
@viccuad I've update the workflows to keep the same approach that we have now. Please, review it again.

Reply:
Thanks! that looks more streamlined for me, resolving!